src/decorator-annotator.ts

```diff
@@ -52,8 +52,8 @@ export class DecoratorClassVisitor {
   /**
    * Determines whether the given decorator should be re-written as an annotation.
    */
-  private shouldLower(decorator: ts.Decorator) {
-    for (const d of getDecoratorDeclarations(decorator, this.typeChecker)) {
+  public static shouldLower(decorator: ts.Decorator, typeChecker: ts.TypeChecker) {
```

Comment:
make this a top level function if it doesn't access state from the class? static class level functions don't really make a lot of sense in TS.